examples/jsm/renderers/nodes/core/InputNode.js

```diff
@@ -0,0 +1,38 @@
+import Node from './Node.js';
+
+class InputNode extends Node {
+
+	constructor( type ) {
+		
+		super( type );
+		
+		this.isInputNode = true;
+		
+		// force constant for now
+		this.constant = true;
```

Comment:
I guess when `constant` is set to `false`, it's a dynamic input and thus a uniform, right?

Reply:
Yes, after implement uniforms I will set `this.constant = false` as default.